avocado/plugins/human.py

```diff
@@ -39,7 +38,7 @@ class Human(ResultEvents):
                       'CANCEL': output.TERM_SUPPORT.CANCEL}
 
     def __init__(self, args):
-        self.log = logging.getLogger("avocado.app")
+        self.log = LOG_UI
```

Comment:
Actually I did this on purpose, because by using the module you might want to get this instance log and not the overall log. (as a prep for multi-job-interfaces where we will probably have to deal with this)